modules/rosa-how-upgrades-work.adoc

```diff
@@ -0,0 +1,45 @@
+
+// Module included in the following assemblies:
+//
+// upgrading/rosa-upgrading-sts.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="rosa-how-upgrades-work_{context}"]
+= How ROSA cluster upgrades work
+
+Upgrades are manually initiated (one-time) or automatically scheduled (recurring). Red Hat Site Reliability Engineers (SREs) monitor upgrade progress and either proactively notify to take any corrective actions or remedy any issues encountered.
+
+[id="rosa-version-operator_{context}"]
+== Cluster Version Operator
+
+The `Cluster Version` Operator (CVO) is the primary component that orchestrates and facilitates the OpenShift Container Platform update process. The scheduling, monitoring, and notifications of ROSA cluster upgrades are orchestrated through the `Managed Upgrade` Operator (MUO). The MUO manages the orchestration of automated in-place cluster upgrades by satisfying the operating conditions that a managed cluster must hold, both pre- and post-invocation of the cluster upgrade.
```

Comment:
We're covering MUO under the title CVO. Perhaps we separate them out or perhaps leave out the title and have it just under 'how rosa cluster upgrades work' because the content is small